Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM alpine:latest
+RUN apk add hugo yarn make
```

Comment:
Before run `apk add hugo yarn make`, firstly run `apk update` is better.
The reason is adding component versions may be old. To keep the newest version is used should update repo first.

Reply:
Great catch!